helm/templates/log-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "schema-registry.fullname" . }}-log-configmap
+  labels:
+    {{- include "schema-registry.labels.standard" . | nindent 4 }}
+data:
+  log4j.properties: |-
+    log4j.rootLogger={{ .Values.logging.level }}, stdout
+
+    log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+    log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+    log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
```

Comment:
Logging class name and line number is costly.
Make this equivalent with the following which is tested and used in pinot. However, please note that the following pattern is from log4j2.

`%d{yyyy-MM-dd HH:mm:ss.SSS} [%level] [%thread] [%c{1.}] - %msg%n`

Reply:
Removing %L should be good then? We anyway seem to log class name